arm-sql/2015-05-01-preview/swagger/virtualNetworkRules.json

```diff
@@ -209,15 +209,15 @@
           "nextLinkName": "nextLink"
         },
         "x-ms-examples": {
-          "List of virtual network rules": {
+          "List virtual network rules": {
             "$ref": "../examples/VirtualNetworkRulesList.json"
           }
         }
       }
     }
   },
   "definitions": {
-    "VnetFirewallRuleProperties": {
+    "VirtualNetworkRuleProperties": {
       "description": "Properties of a virtual network rule.",
       "type": "object",
       "properties": {
```

Comment:
description of `virtualNetworkSubnetId` says `VnetSubnetId`, which contains "vnet" but could also be a little more wordy in general ("The subnet ID of the virtual network rule."?).